go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/cobra v1.1.3
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.21.1
-	k8s.io/apiextensions-apiserver v0.21.1
-	k8s.io/apimachinery v0.21.1
-	k8s.io/client-go v0.21.1
+	k8s.io/api v0.22.0-beta.2
+	k8s.io/apiextensions-apiserver v0.22.0-beta.2
+	k8s.io/apimachinery v0.22.0-beta.2
+	k8s.io/client-go v0.22.0-beta.2
```

Comment:
IIRC, we need to also re-generate the CRD manifests as bumping these dependencies have updated (IIRC description fields?) fields. It looks like this is why the verify check is failing. Likely need to run `make manifests` and/or `make generate` locally and commit the results.